### ISSUE TITLE
System tray icon for Plover shows as generic document instead of Plover logo

### ISSUE THREAD

**Issue by gskaplan** [openstenoproject/plover#1468]
**Describe the bug**
On Windows 11, Plover icon in system tray shows as generic document icon. Plover was installed on new machine with clean version of Windows 11. Clicking the Plover icon has no effect, but right clicking does open a context menu and allows me to open Plover. On my older machine with Windows 10, I believe the Plover icon showed up in the tray.

**To Reproduce**

1. Installed in Windows 11 on two different machines, same result. No special actions taken.

**Expected behavior**

Previously, if my memory serves me right, I could see the Plover icon (with the asterisk).

**Screenshots**
![plover-no-logo](https://user-images.githubusercontent.com/1245198/149644758-dbb2079a-2966-44aa-a11d-e06e5df6319f.png)

**Operating system**

 - OS: Windows 11 Pro, version 21H2, OS build 22000.434
 - Plover Version: 4.0.0.dev10

**Hardware**
Occurs on a Lenovo Carbon X1 notebook and a Lenovo ThinkStation desktop

**Comment by user202729** [comment]
What's the bug? That looks exactly like the intended behavior to me.

(that's a steno key with a X symbol overlaid on top, which signifies "no machine connected")

**Comment by gskaplan** [comment]
Looking at it much more closely, I can see that it is a rounded key with an X. And when I connect to my Stentura, the X turns into something else two vertical bars (when output is disabled) and just a steno key when output is enabled. Because the system tray uses such small icons, it looked like a faded document icon with an X indicating it had a problem. When it turned into the two parallel lines (maybe relating to "pause" media icon), I had to snag the image and blow it up in a graphics editor to see that they are two vertical lines. When I enabled output, it turned into a dark outlined white key. I thought I remembered the Plover system tray icon to have the green asterisk or something that was very obvious to me - maybe my memory is failing. So, I guess it's working as designed. I will close this as a bug, but as a developer (with pretty good eyesight) I would turn it into a possible enhancement just to consider a more obvious connection between the icon and Plover, such as the asterisk on the key. Those other states could be overlaid, but it would be clear that it was Plover. As many times as I looked at it, I thought it was a "page" icon with a small X. Sorry for the erroneous bug report.
